cdap-ui/app/cdap/components/BtnWithLoading/BtnWithLoading.scss

```diff
@@ -29,5 +29,11 @@
       fill: $blue-06;
     }
   }
-
+  &.darker-loading-bars {
+    svg.loading-bar {
+      rect {
+        fill: $blue-01;
```

Comment:
should this use the brand primary color?

Reply:
So this was meant for a darker shade. Brand primary might be problematic if the skinning json has a lighter brand primary. Having it fixed here would help us control the change to this component.